superset/assets/src/setup/setupFormatters.js

```diff
@@ -0,0 +1,35 @@
+import { getNumberFormatter, getNumberFormatterRegistry, createSiAtMostNDigitFormatter, NumberFormats } from '@superset-ui/number-format';
+import { getTimeFormatterRegistry, smartDateFormatter, smartDateVerboseFormatter } from '@superset-ui/time-format';
+
+export default function setupFormatters() {
+  const defaultNumberFormatter = createSiAtMostNDigitFormatter({ n: 3 });
+
+  getNumberFormatterRegistry()
+    .registerValue(defaultNumberFormatter.id, defaultNumberFormatter)
+    .setDefaultKey(defaultNumberFormatter.id)
+    // Add shims for format strings that are deprecated or common typos.
```

Comment:
cc @michellethomas I added shim for any pattern with > 10 occurrences (excluding ones that does not make sense)

Reply:
👍 I'm not sure that we need to add patterns that don't seem like they were valid formats. Up to you though. Thanks for adding this!